oic.r.colour.rgb/colorRGB.raml

```diff
@@ -0,0 +1,68 @@
+#%RAML 0.8
+title: OICColourRGB
+version: v1.1.0-20160519
+documentation:
+  - title: © 2016 Open Connectivity Foundation, Inc. All rights reserved.
+    content: |
+        Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+        1.  Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+        2.  Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
+
+        THIS SOFTWARE IS PROVIDED BY THE Open Connectivity Foundation, INC. "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE OR WARRANTIES OF NON-INFRINGEMENT, ARE DISCLAIMED. IN NO EVENT SHALL THE Open Connectivity Foundation, INC. OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+schemas:
+  - ColourRGB: !include oic.r.colour.rgb.json
+
+traits:
+  - interface:
+      queryParameters:
+        if:
+          enum: ["oic.if.a","oic.if.baseline"]
+
+/ColourRGBResURI:
+  description: |
+    This resource specifies the actual colour in the RGB space represented as a string.
+    Each colour value is described with a Red, Green, Blue component.
+    These colour values are encoded as an array of integer values ([R,G,B]).
+    The minimum and maximum colour value per component is described by the range array.
+    When the range value is omitted, then the range is [0,255].
+  displayName: Colour RGB
+  is: [ interface ] # valid for all methods
+
+  get:
+    description: |
+      Retrieves the current colour in RGB.
+      Value is an array of integer values in the order R,G,B.
+    responses:
+      200:
+        body:
+          application/json:
+            schema: ColourRGB
+            example: |
+              {
+                "rt":       ["oic.r.colour.rgb"],
```

Comment:
>colour [](start = 36, length = 6)

Nit: so do we go wiht the US color here or the British color? Also are there any localization etc requirements at all around these strings -?